src/main/java/org/fundacionjala/coding/kevincristhian/Customer.java

```diff
@@ -0,0 +1,41 @@
+package org.fundacionjala.coding.kevincristhian;
+
+/**
+ * customer class, it has a 1 to N
+ * relation with rental class.
+ */
+public class Customer {
+    private String name;
+    private Rental rental;
```

Comment:
Something is wrong here
Customer should have a List of rentals
List<Rental> rentals;